src/OSmOSE/Spectrogram.py

```diff
@@ -342,6 +341,8 @@ def sensitivity(self):
     @sensitivity.setter
     def sensitivity(self, value):
         """Always assume the sensitivity is given in dB"""
+        if not value:
+            value = 0
```

Comment:
i would put directly 0 as its default value rather than None in that case, and 0 is actually the correct value to put when we do not want to take into account sensitivity_dB